CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Breaking changes result in a different major. UI changes that might break custom
 
 ## [next-version]
 
+## [0.13.0-rc.1]
+
+Install with `npm install onfido-sdk-ui@0.13.0-rc.1`
```

Comment:
any reason why you put this instruction on the changelog? You think it might not be clear otherwise?